js/app.js

```diff
@@ -0,0 +1,83 @@
+
+//question 1
+
+let obj ={
+	insideObj: {
+		key1: 'yo';
+		key2: 12
+	}
+
+}
+
```

Comment:
this is an object that has 2 properties (that are not arrays) inside of another object.  that's not what we wanted.